python/tests/test_aac/context/test_definition.py

```diff
@@ -18,10 +19,88 @@ def test_get_root_key(self):
 
         definition = context.get_definitions_by_name("abstract")
         self.assertEqual(definition[0].get_root_key(), "modifier")
-        
+
+    def test_is_import(self):
+        context = LanguageContext()
+        definition = context.parse_and_load(IMPORT_DEFINITION)[0]
+        self.assertTrue(definition.is_import())
+        definition = context.get_definitions_by_name("Schema")[0]
+        self.assertFalse(definition.is_import())
+
+    def test_get_python_module_name(self):
```

Comment:
Be sure to do fail cases for these as well. You checked both states for `is_import`